src/tools/tools.service.ts

```diff
@@ -569,6 +577,26 @@ function getSystemTools() {
     userDescription:
       'Execute Python code for various tasks, including data analysis, file processing, and visualizations. Supports the installation of any library such as NumPy, Pandas, SciPy, and Matplotlib. Users can create new files or convert existing files, which are then made available for download.'
   });
+  systemTools.set(SystemTools.LLM, {
+    type: ToolType.SYSTEM,
+    id: SystemTools.LLM,
+    createdAt: new Date('2024-12-12'),
+    ...llmTool,
+    inputSchema: llmTool.inputSchema.bind(llmTool),
+    isExternal: false,
+    userDescription:
+      'Give a prompt to an LLM assistant. Useful to extract and re-format information, and answer intermediate questions.'
```

Comment:
I would update the description to reflect the tool's description.

`Uses expert LLM to work with data in the existing conversation (classification, entity extraction, summarization, ...)`